src/extension.ts

```diff
@@ -128,7 +128,7 @@ export async function activate(context: vscode.ExtensionContext) {
     registerCommandWithTelemetry('azure-api-center.openUrl', openUrlFromTreeNode);
     registerCommandWithTelemetry('azure-api-center.apiCenterWorkspace.signInToDataPlane', SignInToDataPlane);
     registerCommandWithTelemetry('azure-api-center.apiCenterWorkspace.refresh', async (context: IActionContext) => ext.dataPlaneTreeItem.refresh(context));
-    registerCommandWithTelemetry('azure-api-center.apiCenterWorkspace.addApis', getDataPlaneApis);
+    registerCommandWithTelemetry('azure-api-center.apiCenterWorkspace.addApis', ConnectDataPlaneApi.getDataPlaneApis);
```

Comment:
A liite confusing: the commnad is [add] `addApis`, while the function is [get] `getDataPlaneApis`?

Reply:
rename to addDataPlaneApis